### ISSUE TITLE
Alt text for survey task filter icons doesn't display when icon is missing

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/Panoptes-Front-End#2913]
Survey task filter icons correctly show alt text when images don't download (see screenshot.)
![image](https://cloud.githubusercontent.com/assets/59547/17931328/d0a60b7e-6a02-11e6-8b7f-823dc218449f.png)

During @mrniaboc and @aliburchard's demo of the survey task today, the filter boxes showed up as blank grey squares if the filters were set up with text but no icon files were uploaded. We should check that the alt text is being set even if the image file doesn't exist.

**Comment by eatyourgreens** [comment]
Test project to reproduce that bug here: https://www.zooniverse.org/projects/mrniaboc/zooniverse-go/classify (Try the 'Hair' filter.)

Looks like the filter button elements are empty if no filter icons were uploaded.

**Comment by aliburchard** [comment]
also, I think it's worth noting that there are two ways for these to be image-less:
1) if no image is specified in the characteristics.csv header because the team doesn't have one (so like=cat/dog instead of like=cat/dog; catdog.svg)
2) if the team has specified an icon image, but hasn't actually uploaded one.

**Comment by eatyourgreens** [comment]
I think `if value.image?` here needs an `else` for the case where the image doesn't exist. Probably something as simple as `else value.label`.
https://help.github.com/articles/syncing-a-fork/

**Comment by simoneduca** [comment]
I can reproduce using the project you linked to @eatyourgreens 
However, I created my own local project (without uploading icons) to test the fix you suggest above and I can still see the alt text.
<img width="725" alt="screen shot 2017-04-07 at 14 34 27" src="https://cloud.githubusercontent.com/assets/8989606/24802815/3080e8ac-1ba1-11e7-9ae5-89a36f766ee6.png">
Thoughts?

**Comment by eatyourgreens** [comment]
@simoneduca Is that what I meant by "the filter boxes showed up as blank grey squares if the filters were set up with text but no icon files were uploaded."? It's so long ago I've forgotten.

**Comment by simoneduca** [comment]
I know, sorry. Here's what you see on zooniverse-go project https://www.zooniverse.org/projects/mrniaboc/zooniverse-go/classify
<img width="534" alt="screen shot 2017-04-07 at 14 56 50" src="https://cloud.githubusercontent.com/assets/8989606/24803263/969db5f6-1ba2-11e7-98be-141ad41fd61d.png">
As far as I can see the set up is the same as my project

**Comment by eatyourgreens** [comment]
Ah right. From memory, the bug is that the code renders an empty `<button>` tag if the image doesn't exist. I'd change it to add the text label.

**Comment by simoneduca** [comment]
That's right, but I don't understand why it seems I can't reproduce it locally (See my first screenshot). I must be doing something wrong.

**Comment by eatyourgreens** [comment]
Could you fix the if block that's missing an else before closing?

**Comment by simoneduca** [comment]
I'm trying to say that your suggested fix doesn't seem necessary, because the bug is due to something else.

**Comment by simoneduca** [comment]
Got it. My bad, I didn't read the previous posts properly. Fix coming.
